geode-connectors/src/main/java/org/apache/geode/connectors/util/internal/MappingConstants.java

```diff
@@ -27,5 +27,8 @@
   public static final String GROUP_NAME = "groups";
   public static final String PDX_CLASS_FILE = "pdx-class-file";
 
+  public static final String THERE_IS_NO_JDBC_MAPPING_ON_PROXY_REGION =
+      "There is no jdbc mapping on proxy region.";
```

Comment:
I still don't understand why we have this message. Why would we say this? Would it be better in this case to not say anything?

Reply:
Since proxies do not have jdbc-mappings it seems confusing that when you run the create jdbc-mapping command you need to add any groups that have the proxies defined in them (correct me if I'm wrong about that). Would it be better if you only told create jdbc-mapping the location of the region you want to add the mapping to, and then our implementation would find any proxy definitions (if !synchronous) and update them with the AEQ id?

Reply:
As discussed, the proxy needs to keep aeq id.